shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -46,6 +46,10 @@ bool AndroidExternalViewEmbedder::SubmitFrame(
   // TODO(egarciad): Implement hybrid composition.
   // https://github.com/flutter/flutter/issues/55270
   TRACE_EVENT0("flutter", "AndroidExternalViewEmbedder::SubmitFrame");
+  if (should_run_rasterizer_on_platform_thread_) {
+    // Don't submit the current frame if the frame will be resubmitted.
+    return false;
```

Comment:
Is the result actually used? Or it is the same with iOS (the result is not used) If the result is not used, and we don't do anything based on a `false` result, I'd suggest to keep it as true.

Reply:
Done. `SubmitFrame` should return `void`. I will change that in a separate PR.